sdk/storage_datalake/src/properties.rs

```diff
@@ -54,8 +55,9 @@ impl TryFrom<&HeaderMap> for Properties {
     fn try_from(headers: &HeaderMap) -> Result<Self, Self::Error> {
         let header_value = headers
             .get(PROPERTIES)
```

Comment:
Hmm, may be we need to rename to `.try_get` and add a `get` that errors like this.